### PR TITLE
Add GitHub releases and tag version integration

### PR DESCRIPTION
**Pull request by PepijnSenders**

### CODE REVIEW

File: ci/git-release.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env node
```
Comment (nik-kor) [inline ci/git-release.sh]: I think it's better to use .js extension for it

Comment (PepijnSenders) [inline ci/git-release.sh]: Right, forgot to change that 😄

**Comment by nik-kor** [comment]
+1

**Comment by nik-kor** [comment]
+1

**Comment by renannfortes** [comment]
+1

**Pull request merged**